### PR TITLE
shrink operator

### PR DESCRIPTION
**Pull request by brightening-eyes**

### CODE REVIEW

**Comment by nihui** [comment]
hi, please add shrink entry in operators.md

**Comment by brightening-eyes** [comment]
done.
conflicts resolved and added shrink to ops list

File: docs/developer-guide/operators.md [hunk 1]
```diff
@@ -1675,6 +1676,20 @@ else        y = x * lambda
...
+# Shrink
+```
+y = x - bias , x > lambd
+y = x + bias,     x < -lambd
+```
+
+| param id  | name          | type  | default   | description       |
+| --------- | ------------- | ----- | --------- | ----------------- |
+| 0         | bias         | float | 0.0f|                  |
+| 1         | lambd        | float | 0.5f|                 |
```
Comment (nihui) [inline docs/developer-guide/operators.md:1688]: align the formula and table border

**Comment by brightening-eyes** [comment]
done I suppose.

**Comment by nihui** [comment]
图片

**Comment by brightening-eyes** [comment]
@nihui 
I've tried to fix that doc. if there is anything left, please let me know. (my screen reader reads the tables correctly).
thanks.

**Comment by nihui** [comment]
fixed for you anyway  :smiley:

**Comment by brightening-eyes** [comment]
thanks a lot.

File: tests/test_shrink.cpp [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+static int test_shrink_1()
+{
+    return 0
+           || test_shrink(RandomMat(5, 7), 3.4f, 0.3f)
+           || test_shrink(RandomMat(7, 9), 3.1f, 4.0f)
+           || test_shrink(RandomMat(3, 5), 2.0f, 4.0f);
+}
+
```
Comment (nihui) [inline tests/test_shrink.cpp:50]: need 1d and 4d tensor test here

**Comment by nihui** [comment]
Thanks for your contribution !

**Pull request merged**